app/assets/stylesheets/media-show-page.css

```diff
@@ -30,22 +30,11 @@
   width: 125px;
 }
 
-a.streaming-provider {
-  display: block;
-  font-size: 0.9rem;
-  line-height: 1rem;
-}
-
 .manage-screenings .material-symbols-outlined.edit-note {
   position: relative;
   top: 7px;
 }
 
-.txt-sm.availability {
-  font-size: 10px;
-  margin-top: 20px;
-}
-
```

Comment:
Replaced this with a simple utility class that uses tailwind specs: https://tailwindcss.com/docs/font-size